utils/create_validate_pr_quickstarts.ts

```diff
@@ -73,10 +102,17 @@ export const createValidateQuickstarts = async (
     )
     .flatMap((filePath) => {
       if (QUICKSTART_CONFIG_REGEXP.test(filePath)) {
-        return new Quickstart(filePath);
+        return new Quickstart(
+          filePath,
+          undefined, // Note: We are ignoring the `basePath` argument here
```

Comment:
why are we ignoring the `basePath` argument here?

Reply:
Thinking about this more, I could see making the `basePath` and `quickstartContext` arguments an `options` object since they are both optional.

Reply:
> why are we ignoring the `basePath` argument here?
It was just a convenience thing to rely on the default. 

I think an options object is a good way to go. Consider it changed!

Reply:
Addresed in commit https://github.com/newrelic/newrelic-quickstarts/pull/1649/commits/fbde806738e80eca9a0ff4b8615fb0c92d70e3a5